### PR TITLE
WIP fix ramfs

### PR DESCRIPTION
**Pull request by hugelgupf**
Signed-off-by: Christopher Koch <c@chrisko.ch>

### CODE REVIEW

**Comment by rminnich** [comment]
This needs a much better comment :-)
And the cpio fix should be its own thing, right?

File: pkg/cpio/fs_unix.go [hunk 1]
```diff
@@ -187,7 +187,17 @@ func inode(i Info) (Info, bool) {
 }
 
 func GetRecord(path string) (Record, error) {
-	fi, err := os.Lstat(path)
+	return GetResolvedRecord(path, false)
+}
+
+func GetResolvedRecord(path string, resolveSymlinks bool) (Record, error) {
```
Comment (rminnich) [inline pkg/cpio/fs_unix.go]: Sorry, I'm slow this morning, why do we need to resolve symlinks when in many cases that's the wrong thing to do ... what's the use case here?

File: pkg/uroot/source.go [hunk 2]
```diff
@@ -24,6 +24,10 @@ func SourceBuild(opts BuildOpts) (ArchiveFiles, error) {
 
 	log.Printf("Collecting package files and dependencies...")
 	deps := make(map[string]struct{})
+
+	// Always include the source for installcommand.
+	opts.Packages = append(opts.Packages, "github.com/u-root/u-root/cmds/installcommand")
```
Comment (rminnich) [inline pkg/uroot/source.go]: good catch.

Comment (rminnich) [reply pkg/uroot/source.go]: If it's already there it won't get in twice, right?

**Comment by rminnich** [comment]
what's the story on this one? keep or not?

**Comment by rjoleary** [comment]
Is there anything still valuable in here, or can it be closed?

**Pull request closed without merging**